qiskit/providers/aer/extensions/snapshot.py

```diff
@@ -0,0 +1,137 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+Simulator command to snapshot internal simulator representation.
+"""
+
+import warnings
+
```

Comment:
Are these files a duplication of what is in `qiskit/providers/aer/aer_snapshots.py`? I think this is a better way to lay them out so I would suggest removing that file and keeping these ones (above comments still apply to these files :) )

Reply:
Fixed 😄